app/audit/AuditEvents.scala

```diff
@@ -16,88 +16,9 @@
 
```

Comment:
There's no details about auditing / logging in the ticket.  There will need to be separate tickets for that as this ticket will get too big otherwise.   I've highlighted these will be needed and I'm not going to second guess it as its likely whatever is done now will neeed to be re-done anyway.
Keeping the existing auditing wasnt an option either as the way it was implemented was a bit messy and the audit events didnt make sense in the new context